conans/__init__.py

```diff
@@ -3,14 +3,13 @@
 from conans.model.conan_file import ConanFile
 from conans.model.options import Options
 from conans.model.settings import Settings
-from conans.client.cmake import CMake
-from conans.client.meson import Meson
-from conans.client.gcc import GCC
-from conans.client.configure_environment import ConfigureEnvironment
-from conans.client.configure_build_environment import (AutoToolsBuildEnvironment, VisualStudioBuildEnvironment)
+from conans.client.build.cmake import CMake
+from conans.client.build.meson import Meson
+from conans.client.build.gcc import GCC
+from conans.client.build.configure_environment import ConfigureEnvironment
+from conans.client.build.configure_build_environment import (AutoToolsBuildEnvironment, VisualStudioBuildEnvironment)
```

Comment:
Split in two.

Reply:
What do you mean?

Reply:
One for ``visual_studio`` and different one for ``autotools``